theme/templates/email/signup_verify.txt

```diff
@@ -0,0 +1,8 @@
+{% load i18n %}
+{% block main %}
+{% trans "Welcome to HydroShare. This email address was used to request an account on HydroShare." %}
+{% trans "If you originated the request, please use the link below to verify your email address and activate your account." %}
+http://{{ request.get_host }}{{ verify_url }}
```

Comment:
@ironfroggy @mjstealey @selimnairb another option is to change this to https so https is in the link to be clicked for account verification in the verification email, which will work if USE_SSL is set to true. In the local development environment when USE_SSL is set to False, developers can manually change https to http in the URL link and it will work as expected with account being verified. This sounds like the best solution I can see. If I don't hear objections, that is what I will do.